ads/_config.js

```diff
@@ -747,3 +747,14 @@ export const adConfig = {
   },
 
 };
+
+
+/**
+ * Specifies which networks can share master frames.
+ * @type {!Object<string, !Array<string>>}
+ */
+export const masterFrameAccessibleTypes = {
+  adsense: ['doubleclick'],
```

Comment:
instead of dynamically compute the string, why not just assign a fixed string?
```
export const masterFrameNameOverride = {
  adsense: 'google_network',
  doubleclick: 'google_network',
  admob: 'google_network',
}
```

This is way easier and less error-prone than
```
export const masterFrameAccessibleTypes = {
   adsense: ['doubleclick', 'admob'],
   doubleclick: ['adsense', 'admob'],
   admob: ['doubleclick', 'adsense'],
```